extension.toml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+api = "0.8"
+
+[extension]
+  description = "A Cloud Native Buildpacks Image Extension that provides/does nothing. For testing only."
+  homepage = "https://github.com/paketo-buildpacks/pipeline-builder-stool-pigeon"
+  id = "paketo-buildpacks/pipeline-builder-stool-pigeon"
+  keywords = ["nothing"]
+  name = "Paketo Image Extension for Pipeline Builder Stool Pigeon"
+  version = "{{.version}}"
+
+  [[extension.licenses]]
+    type = "Apache-2.0"
+    uri = "https://github.com/paketo-buildpacks/pipeline-builder-stool-pigeon/blob/main/LICENSE"
+
+[metadata]
+  include-files = ["LICENSE", "NOTICE", "README.md", "bin/generate", "bin/detect", "bin/main", "extension.toml"]
+  pre-package = "scripts/build.sh"
+
+  [[metadata.dependencies]]
+    cpes = ["cpe:2.3:a:apache:maven:3.9.3:*:*:*:*:*:*:*"]
+    id = "maven"
+    name = "Apache Maven"
+    purl = "pkg:generic/apache-maven@3.9.3"
+    sha256 = "e1e13ac0c42f3b64d900c57ffc652ecef682b8255d7d354efbbb4f62519da4f1"
+    stacks = ["io.buildpacks.stacks.bionic", "io.paketo.stacks.tiny", "*"]
+    uri = "https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.9.3/apache-maven-3.9.3-bin.tar.gz"
+    version = "3.9.3"
+
+    [[metadata.dependencies.licenses]]
+      type = "Apache-2.0"
+      uri = "https://www.apache.org/licenses/"
+
+[[stacks]]
+  id = "io.buildpacks.stacks.bionic"
+
+[[stacks]]
+  id = "io.paketo.stacks.tiny"
```

Comment:
These are not needed with the wildcard stack. They're only in the other buildpacks for historical reasons. We could remove them from there also, but just haven't gotten around to it.

Reply:
agreed.. legacy here as I based this project from canary =) I'll remove from here, given the window during which both stacks & extensions co-existed will be small historically.. not worth immortalizing the idea this was ever normal =)

Reply:
Removed.